src/link/filelink.cpp

```diff
@@ -34,9 +34,8 @@ void FileLink::_writeData(const QByteArray& data)
 
     // This save the data as a structure to deal with the timestamp
     if(_openModeFlag == QIODevice::WriteOnly && _file.isWritable()) {
-        QString time =  QTime::fromMSecsSinceStartOfDay(_timer.elapsed()).toString(_timeFormat);
-        Pack pack{time, data};
-        _inout << pack.time << pack.data;
+        _inout << QTime::fromMSecsSinceStartOfDay(_timer.elapsed()).toString(_timeFormat)
```

Comment:
The usage of the structure here is less error prone, you are creating the structure by hand here, and this does not sync automatically with the Pack definition, resulting in a less readable and more error prone approach.

Reply:
ok, I'll ignore this patch.